theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx

```diff
@@ -0,0 +1,508 @@
+
+`implementation 'com.theoplayer.theoplayer-sdk-android:core:8.0.0'`
+
+## Setting up the license
+
+You need to configure your THEOplayer license, if you don't enter this license string, you might encounter licensing issues (e.g.: when trying to play non-whitelisted video streams).
+
+First, obtain a THEOplayer Android SDK license through the [THEOplayer Developer Portal](https://portal.theoplayer.com/login).
+![](https://cdn.theoplayer.com/images/git/theoplayer-android-sdk-license-string.png)
+
+Then, identify your license string from the screenshot above and add the `<meta-data ...` tag below at the appropriate location in your `AndroidManifest.xml`, and swap `your_license_here` with your
+actual license string.
+
+```xml
+<application>
+    <meta-data
+        android:name="THEOPLAYER_LICENSE"
+        android:value="your_license_here" />
+</application>
+```
+
+Additionally, to play online videos with THEOplayer, you first need to add [INTERNET](https://developer.android.com/reference/android/Manifest.permission.html#INTERNET) permission into the `AndroidManifest.xml` file too.
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+More information regarding the license can be found [here](../../../how-to-guides/12-license/00-introduction.md).
+
+## Add THEOplayer view to your application
+
+THEOplayer can be added to an Android application either:
+
+1. **Through a layout (XML)**: By adding the `THEOplayerView` to your layout XML file (e.g., `activity_main.xml`), or
+2. **Programmatically using the Constructor API**: This approach also allows you to configure the license string through a programmatic configuration instead of the `AndroidManifest.xml`.
+
+### Adding THEOplayerView via XML layout
+
+You can add the `THEOplayerView` to the layout of the `MainActivity` with the snippet below.
+
+![](../../../../theoplayer/assets/img/android-getting-started-13_2.png)
+
+At least the `id`, `width` and `height` attributes need to be specified on the new view. The `id` is important as it is used to identify the view object in your `MainActivity` class.
+
+```xml
+<com.theoplayer.android.api.THEOplayerView
+    android:id="@+id/theoplayer"
+    android:layout_width="match_parent"
+    android:layout_height="300dp"
+    app:layout_constraintTop_toTopOf="parent">
+</com.theoplayer.android.api.THEOplayerView>
+```
+
+![](../../../../theoplayer/assets/img/android-getting-started-14_2.png)
+
+
+### Adding THEOplayerView programmatically (Constructor API)
+
+In the Activity/Fragment, you can use the [constructor API](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerView.html) to create a THEOplayer instance
+by passing along a context and a [`THEOplayerConfig`](https://docs.theoplayer.com/api-reference/android/index.html?com/theoplayer/android/api/THEOplayerConfig.html).
+
+In this `THEOplayerConfig`, you must configure your license string, as demonstrated in the snippet below.
```

Comment:
"must" is wrong in this context.
Users have the option to either set the license in the AndroidManifest or through the THEOplayerConfig API. Even when using the constructor, they could set the license in the AndroidManifest.